### PR TITLE
feat(base): add tests for git and gitlfs

### PR DESCRIPTION
**Pull request by ghuntley**
## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/6598

## Release Notes

```release-note
```

### CODE REVIEW

File: base/Dockerfile [hunk 1]
```diff
@@ -37,9 +37,12 @@ ENV LANG=en_US.UTF-8
 RUN upgrade-packages
 
 ### Git ###
+LABEL dazzle/test=tests/tool-git.yaml
 RUN add-apt-repository -y ppa:git-core/ppa \
     && install-packages git git-lfs
 
+RUN git lfs install --system
```
Comment (ghuntley) [inline base/Dockerfile:44]:
``` ---> Running in 2d49b0ee581c
Error: Failed to call git rev-parse --git-dir: exit status 128 
Git LFS initialized.
```

File: base/tests/tool-git.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+- desc: git should be installed
+  command: [which, dsadsada]
```
Comment (ghuntley) [inline base/tests/tool-git.yaml:2]: Undo this.

Comment (ghuntley) [reply base/tests/tool-git.yaml:2]: Bug: tests on the base image is not working or configured correctly. This test case should have failed.

**Comment by princerachit** [comment]
This looks outdated. Shall I close this PR and delete the branch @ghuntley ?

**Comment by Furisto** [comment]
Closing. Feel free to open again @ghuntley if you still would like to work on this.

**Pull request closed without merging**